src/panel/direction/RouteResult.jsx

```diff
@@ -73,7 +74,21 @@ export default class RouteResult extends React.Component {
     if (this.props.error) {
       return <div className="itinerary_no-result">
         <span className="icon-alert-triangle" />
-        <div>{_('Qwant Maps found no results for this itinerary.')}</div>
+        <div>{
+          this.props.errorCode >= 500 && this.props.errorCode < 600
+            ? _('The service is temporarily unavailable, please try again later.', 'direction')
+            : _('Qwant Maps found no results for this itinerary.', 'direction')
+        }</div>
+        {
+          this.props.vehicle === 'publicTransport' &&
+          <div>{
+            _(
+              'For information, public transport mode is currently in test version for some ' +
```

Comment:
We had some discussion about that sentence in the Jira ticket. See with @Charlotte-Ziegler what is the chosen wording.